packages/spectral/src/serverTypes/convertComponent.ts

```diff
@@ -286,7 +286,7 @@ export const convertConnection = ({
 
   const convertedInputs = Object.entries(inputs).map(([key, value]) => {
     if ("templateValue" in value) {
-      return convertTemplateInput(key, value, inputs);
+      return convertTemplateInput(key, value as ConnectionTemplateInputField, inputs);
```

Comment:
What necessitates the cast? Are we able to update `convertTemplateInput` to make the types make sense?